Plugin/API.cs

```diff
@@ -16,6 +16,16 @@ public static bool alwaysUpdate
                 Settings.fetch.AlwaysUpdate = value;
             }
         }
+        public static double? getEndTime()
+        {
+            foreach (var patch in Trajectory.fetch.patches)
```

Comment:
According to this: http://wiki.kerbalspaceprogram.com/wiki/Garbage_Reduction#avoid_foreach

It would be better to not have foreach loops in KSP. I'm not quite sure if this applies here. What do you think?